projects/rocprofiler-compute/src/argparser.py

```diff
@@ -255,7 +258,31 @@ def omniarg_parser(
         nargs="+",
         dest="dispatch",
         required=False,
-        help="\t\t\tDispatch ID filtering.",
+        help=(
+            "\t\t\tWhich dispatch iterations of the kernel to filter \n"
+            "\t\t\t(e.g. 1 3:5 captures 1st, 3rd, 4th and 5th iterations)."
+        ),
+    )
+    profile_group.add_argument(
+        "--iteration-multiplexing",
+        type=str,
+        dest="iteration_multiplexing",
+        metavar="",
+        required=False,
+        nargs="?",
+        choices=[
+            "simple",
+            "kernel",
+            "launch",
+        ],
+        const="launch",
+        help=(
+            "\t\t\tChoose the iteration multiplexing method: (DEFAULT: launch).\n"
+            "\t\t\t   simple (i.e. Round robin over all kernel dispatches\n"
+            "\t\t\t   kernel (i.e. Round robin over unique kernel dispatches)\n"
```

Comment:
I'd suggest not using "dispatches" here. Think about the common SW users, they typically call this kernel calls or kernel invocations.